.env.dev

```diff
@@ -1,3 +1,3 @@
 APP_CONFIG_DEFAULT_SOURCE=gnnuzabk
 APP_UNSAFE_ALLOW_DYNAMIC_CONFIG=1
-APP_API_BASE_URL=https://cdn.jwplayer.com
+APP_API_BASE_URL=https://cdn.jwplayer.com
```

Comment:
I suggest making sure the 'newline at end of files' is enabled in your IDE of choice:
- https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file
- https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Let's make sure we sync up our ide settings on all the little formatting things like this to avoid unnecessary noise.

Reply:
Hmm, `the newline at end of files` is enabled, i guess it does not work for env files. Will fix that